tests/venv/postBuild/README.rst

```diff
@@ -5,3 +5,15 @@ It is possible to run scripts after you've built the environment specified in
 your other files. This could be used to, for example, download data or run
 some configuration scripts. For example, this will download and install a
 Jupyter extension.
+
+.. note::
+
+   This file needs to be executable in order to work with ``repo2docker``. If
+   you're on a UNIX operating system, run::

```

Comment:
Can we explicitly say 'macOS or Linux'? A lot of people running macOS probably don't think of their computers as Unix... Sorry to be nitpicky!

Reply:
how bout now?